docs/README.md

```diff
@@ -3,20 +3,10 @@
 zenoh-c API documentation is available on [Read the Docs](https://zenoh-c.readthedocs.io/en/latest/index.html).
 
 -------------------------------
-## How to build it 
+## How to build it
 
-  -- Ubuntu -- 
-
-  ```bash
-  $ cd docs
-  $ sphinx-build -b html . _build/html
-  ```
-
-  -- MacOS -- 
-
-  Update conf.py *Config.set_library_file* with the path to your *libclang.dylib* file.
-
-  ```bash
-  $ cd docs
-  $ sphinx-build -b html . _build/html
-  ```
\ No newline at end of file
+```bash
+doxygen Doxyfile
```

Comment:
I believe the order should be the opposite: first cd to docs, then doxygen  (no need to specify Doxyfile since it should be picked by default)

Reply:
Thanks. I will fix this.